I also get undefined symbols starting with: `arraytypes.obj : error LNK2001: unresolved external symbol cblas_zdotu_sub`.

I don't get these errors if I specify BLIS in the `blas` section of `site.cfg` instead:

```
[blas]
blas_libs = numpy-blis-reference
library_dirs = c:\code\blis\test\lib
include_dirs = c:\code\blis\test\include
```

--- a/site.cfg.example
+++ b/site.cfg.example
@@ -130,6 +130,21 @@
 # include_dirs = /opt/OpenBLAS/include
 # runtime_library_dirs = /opt/OpenBLAS/lib
 
+# BLIS
+# ----
+# BLIS (https://github.com/flame/blis) also provides a BLAS interface.  It's a
+# relatively new library, its performance in some cases seems to match that of
+# MKL and OpenBLAS, but it hasn't been benchmarked with Numpy or Scipy yet.
+#
+# Notes on compiling BLIS itself:
+#   - the CBLAS interface (needed by Numpy) isn't built by default; define
+#     BLIS_ENABLE_CBLAS to build it.
+#   - ``./configure auto`` doesn't support 32-bit builds, see gh-7294 for
+#     details.
+# Notes on compiling Numpy against BLIS:
+#   - ``include_dirs`` below should be the directory where the BLIS cblas.h
+#     header is installed.
+#
 # [blis]
 # libraries = blis
 # library_dirs = /home/username/blis/lib

